<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>120</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TABELA DE CADASTRO</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="caixa_movimentacao">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>200</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 7px;</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="caixa_entrada">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>240</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">alternate-background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
border-radius: px7;</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="caixa_saida">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>280</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">alternate-background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
border-radius: px7;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="caixa_descricao">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>310</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 7px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>218</x>
        <y>200</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tipo de movimentação</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>240</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>entrada</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>280</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>saida</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>320</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>descrição</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>370</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>valor</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="caixa_valor">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>370</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: px7;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="caixa_categoria">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>420</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentText">
       <string>New Item</string>
      </property>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>salario</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Aluguel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>despesas </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>pagamentos</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>420</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>categoria</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="caixa_datamovimentacao">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>480</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">alternate-background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
border-radius: px7;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>480</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Data da movimentação</string>
      </property>
     </widget>
     <widget class="QPushButton" name="caixa_botao">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>550</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>Cadastrar</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
